redis-cluster/base/init-job.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-init-scripts
+data:
+  cluster-init.sh: |
```

Comment:
discussion/for the future: what do you think about moving these out to their own files, then adding them via `configMapGenerator`? It feels a bit more discoverable to me to have them in their own files, rather than just yaw YAML strings

Reply:
Funny you should say that I played around with it in separate files but felt this more discoverable as it was close to the resource using it. But Im not attached i can change it.